config.example.yml

```diff
@@ -38,28 +38,25 @@ workload_distribution_pipeline:
     results_folder: "azureml://FOLDER_PATH_WHERE_STORE_WORKLOAD_FILES"
 
 inference_pipeline:
-  customers:
-    - name: customer1
-      container_root: "azureml://subscriptions/{subscription}/resourcegroups/{resourcegroup}/workspaces/{workspace}/datastores/{datastore_name}/paths/"
-      model_parameters:
-        imgsz: 4000
-        save_txt: True
-        exist_ok: True
-        save_csv: True
-        skip_evaluation: True
-        save_blurred_image: True
-        batch_size: 4
-      inputs:
-        files_to_blur: "azureml://subscriptions/{subscription}/resourcegroups/{resourcegroup}/workspaces/{workspace}/datastores/{datastore_name}/paths/files_to_blur.txt"
-      outputs:
-        results_path: "azureml://subscriptions/{subscription}/resourcegroups/{resourcegroup}/workspaces/{workspace}/datastores/{datastore_name}/paths/"
+  customer_name: data_office
+  container_root: "azureml://subscriptions/{subscription}/resourcegroups/{resourcegroup}/workspaces/{workspace}/datastores/{datastore_name}/paths/"
+  model_parameters:
+    imgsz: 4000
+    save_txt: True
+    exist_ok: True
+    skip_evaluation: True
+    save_blurred_image: True
+    batch_size: 4
+  inputs:
+    files_to_blur: "azureml://subscriptions/{subscription}/resourcegroups/{resourcegroup}/workspaces/{workspace}/datastores/{datastore_name}/paths/batch_1.txt"
```

Comment:
batch_1.txt We dont want to hardcode the file, this needs to be handled in the Python code